crates/miden-tx/src/host/mod.rs

```diff
@@ -343,6 +348,12 @@ impl<'store, 'auth, A: AdviceProvider> TransactionHost<'store, 'auth, A> {
 
         let slot_index = slot_index.as_int() as u8;
         if new_map_value != prev_map_value {
```

Comment:
Not related to this PR, but we should update the expected stack state on line 307 above.